_data/people.yaml

```diff
@@ -162,17 +168,63 @@
 
 # Master students
 
+
+
+
+# Bachelor students
+
+# Alumni
+
+- name: Jibiana Jakpor
+  position: bachelor
+  start: 2024
+  end: 2024
+
+- name: Benjamin Przybocki
+  position: bachelor
+  start: 2023
+  end: 2024 
+  now: Oxford
+
+- name: Kaia Xiaofu Li
+  position: bachelor
+  start: 2023
+  end: 2024 
+
+
+- name: Ying Sheng
+  position: phd
+  img: sheng.jpg
+  website: https://profiles.stanford.edu/ying-sheng
+  start: 2018
+  end: 2024
+  now: UCLA
```

Comment:
So, Ying is currently at XAI, and will start at UCLA in a couple of years.  Maybe just put both?